src/sagemaker_mxnet_container/serving.py

```diff
@@ -157,7 +161,7 @@ def default_input_fn(self, input_data, content_type):
         """
         if content_type in self.VALID_CONTENT_TYPES:
             np_array = encoders.decode(input_data, content_type)
-            return mx.nd.array(np_array)
+            return mx.nd.array(np_array).as_in_context(_context())
```

Comment:
Nice!

I'm not 100% certain this will work with the EI context.

Reply:
ran the MXNet + EI integ test in the Python SDK against this change, and it passed

Reply:
Did you create a container and pass it as a parameter for a MXNet estimator? Any chance you can paste the output or showing that the data you requested was in numpy?